cmd/all-in-one/all_in_one_test.go

```diff
@@ -72,11 +74,32 @@ func healthCheck(t *testing.T) {
 		},
 		10*time.Second,
 		time.Second,
-		"expecting query endpoint to be healhty",
+		"expecting query endpoint to be healthy",
 	)
 	t.Logf("Server detected at %s", queryAddr)
 }
 
+func healthCheckV2(t *testing.T) {
+	if os.Getenv("HEALTHCHECK_V2") == "false" {
+		t.Skip("Skipping health check for V1 Binary")
```

Comment:
To what scope does t.Skip apply?

Reply:
Applies to V1 scope.
I have exposed 13133 port in Dockerfile and also configured yaml file to observe 0.0.0.0:13133.
But still the error is logged as connection refused?

Reply:
you missed another place - [f863d0a](https://github.com/jaegertracing/jaeger/pull/5831/commits/f863d0aed51d94a9d347f00d6e15151105699b3c)

Reply:
`t.Skip` aborts the whole test.

Reply:
I replaced it with `t.Run("healthCheckV2", healthCheckV2)`, which creates a sub-test which is ok to skip as a whole

Reply:
Ohhh okay, Thank you.
I will get started on other config files as well.